src/nxt_http_route.c

```diff
@@ -1594,8 +1626,15 @@ nxt_http_route_match(nxt_task_t *task, nxt_http_request_t *r,
     nxt_http_route_match_t *match)
 {
     nxt_int_t              ret;
+    nxt_http_action_t      *action;
```

Comment:
Forgot to remove this now it's not needed!?

Reply:
Fixed.